pages/content-ui/script/entryPoints.ts

```diff
@@ -0,0 +1,52 @@
+import { readdirSync, existsSync, mkdirSync, writeFileSync, Dirent, readFileSync, renameSync } from 'fs';
+import { resolve } from 'node:path';
+
+const rootDir = resolve(__dirname, '..');
+const srcDir = resolve(rootDir, 'src');
+
+export const getEntryPoints = (options: { include?: string[]; exclude?: string[]; createRecursively?: boolean }) => {
+  const entryPoints: Record<string, string> = {};
+
+  const foldersToProcess =
+    options.include ||
+    readdirSync(srcDir, { withFileTypes: true })
+      .filter((item): item is Dirent => item.isDirectory())
+      .map(item => item.name);
+
+  foldersToProcess.forEach(folderName => {
+    const shouldExclude = options.exclude && options.exclude.includes(folderName);
+
+    if (!shouldExclude) {
+      const entryFolder = resolve(srcDir, folderName);
+      const entryFileTS = resolve(entryFolder, 'index.ts');
+      const entryFileTSX = resolve(entryFolder, `${folderName}.tsx`);
+      const existingEntryFileTSX = resolve(srcDir, 'App.tsx');
```

Comment:
The details of creating an existing App.tsx as an entry point for pages is a very specific implementation that is difficult to understand from the outside.

App.tsx is not currently used anywhere, so it seems like a good idea to rename it to something like Template.tsx.

Reply:
ahhh i just added that, cause user can choose between just single page or multi page, if user chooses to have multipage, he can always just add stuff into getentrypoints.includes which will generate those template for them. so its easier for them to understand.